### PR TITLE
Added extraSecretMounts option for TLS support

### PR DESCRIPTION
**Pull request by arpitjindal97**

### CODE REVIEW

**Comment by arpitjindal97** [comment]
@naseemkullah  Can u check pr build logs? I wasn't able to find any issue here.

File: charts/jaeger/values.yaml [hunk 1]
```diff
@@ -167,11 +179,17 @@ collector:
...
-  extraConfigmapMounts: []
+  extraSecretMounts: []
   # - name: jaeger-tls
   #   mountPath: /tls
   #   subPath: ""
-  #   configMap: jaeger-tls
+  #   secretName: jaeger-tls
+  #   readOnly: true
+  extraConfigmapMounts: []
+  # - name: jaeger-config
```
Comment (naseemkullah) [inline charts/jaeger/values.yaml]: what's this commented out configmount example, when would we use this?

Comment (arpitjindal97) [reply charts/jaeger/values.yaml]: Collector and agent both has an option of passing config-file

https://www.jaegertracing.io/docs/1.16/cli/

**Comment by naseemkullah** [comment]
Thanks @arpitjindal97

**Pull request merged**